model-optimizer/mo/front/kaldi/loader/loader.py

```diff
@@ -497,3 +498,25 @@ def parse_specifier(string, graph, layer_node_map):
     elif spec == b'ReplaceIndex':
         node = parse_specifier(args[0], graph, layer_node_map)
         return node
+    elif spec == b'Scale':
+        node_name = parse_specifier(args[1], graph, layer_node_map)
+        scale_value = float(args[0])
+        layer_name = 'Scale_{}_{}'.format(node_name, scale_value)
+
+        if layer_name not in layer_node_map:
+            scale_name = graph.unique_id(prefix=layer_name)
+            scale_node = Mul(graph, {'name': scale_name}).create_node()
+
+            layer_node_map[layer_name] = scale_name
+
+            scale_const_name = 'Scale_{}_const_{}'.format(node_name, scale_value)
```

Comment:
Please, use the following naming convention for the node names `<prefix_node_name>/<additional_name>` everywhere. So here it should be `'{}/Scale/{}'.format(node_name, scale_value)`. 

BTW, do we really need the scale value to be the part of the node name?

Reply:
Agreed. `Scale` is redundant in the name. Removed.